### PR TITLE
Updated styles to use Bootstrap

### PR DESCRIPTION
**Pull request by n1k0**
WiP. Following the release of rjsf v0.20.0, this patch updates the HTML semantics to use Bootstrap, and also improves the general styling a little.

We'll move to using some Bootswatch theme after this lands.

/cc @magopian

### CODE REVIEW

**Comment by n1k0** [comment]
r=? @magopian

File: css/rjsf.css [hunk 1]
```diff
@@ -1,96 +0,0 @@
...
-
-.rjsf .field-array-of-string input[type=text] {
-  width: calc(100% - 38px);
-}
-
-.rjsf .field-array-of-string .array-item-remove {
-  float: right;
-  width: 30px;
-  margin-top: -2.3em;
-}
```
Comment (magopian) [inline css/rjsf.css]: so much :heart:

File: package.json [hunk 2]
```diff
@@ -18,9 +18,9 @@
     "dist"
   ],
   "engineStrict": false,
-  "engines" : {
-      "npm": "^2.14.7",
-      "node": ">=4"
```
Comment (magopian) [inline package.json]: my bad, sorry

File: scripts/components/CollectionList.js [hunk 3]
```diff
@@ -133,9 +135,11 @@ function ListActions(props) {
   return (
     <p className="actions">
       <button type="button"
-        className="btn-sync"
+        className="btn-sync btn btn-info"
         onClick={onSyncClick}>Synchronize</button>
-      <LinkButton label="Add" to={`/collections/${name}/add`} />
+      <span> </span>
```
Comment (magopian) [inline scripts/components/CollectionList.js]: are those two empty SPANs needed? They look a bit weird.

Comment (n1k0) [inline scripts/components/CollectionList.js]: Yeah, need to remove them and style the spacing between elements using CSS here.

File: package.json (hunk 2 above)
Comment (n1k0) [inline package.json]: It's not you. 







It's vim.

**Comment by n1k0** [comment]
I'll take this as an r+ :p

**Pull request merged**